handlers/sql_generic_v3/process/node.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<taskDefinition id="sql_generic" name="SQL Generic Command" schema_version="1.0" version="3">
+    <author>support@kineticdata.com</author>
+    <description>
+        Execute a SQL Generic Command.
+    </description>
+    <helpurl>http://community.kineticdata</helpurl>
+    <visible>false</visible>
+    <deferrable>false</deferrable>
+    <parameters>
+        <parameter id="error_handling" label="Error Handling" required="true" tooltip="Determine what to return if an error is encountered." menu="Error Message,Raise Error">Error Message</parameter>
+		<parameter id="jdbc_database" label="Database" required="true" tooltip="Database to Access for example sqlserver, oracle or postgresql.">sqlserver</parameter>

```

Comment:
Add a "menu" attribute with the available options. This will ensure only the supported database types are entered, and also ensure the string values are what you are expecting. You are downcasing the value when checking which jdbc url to build up, but not downcasing when using the value in the jdbc url.

`menu="sqlserver,oracle,postgresql"`

I don't think it makes sense to set a default value for this parameter.